src/pcsr/PCSR.cpp

```diff
@@ -12,30 +12,27 @@
 #include <tuple>
 #include <shared_mutex>
 #include <atomic>
+#include <iostream>
+#include <numa.h>
 
 using namespace std;
 
 // find index of first 1-bit (least significant bit)
 static inline int bsf_word(int word) {
-  int result;
-  __asm__ volatile("bsf %1, %0" : "=r"(result) : "r"(word));
-  return result;
+    int result;
```

Comment:
Are you using a code formatter? If so, what style did you decide to use?

Reply:
I used the built-in code formatter of CLion. Probably it would be good to add a config file for code formatting. Often I use this one:
https://github.com/AutoPas/AutoPas/blob/master/.clang-format

Reply:
Yes, we should impose a global style and keep everything consistent within one style. Further style improvements/fixes could be done while cleaning the C-style code.